core/src/main/java/org/mskcc/cbio/portal/servlet/QueryBuilder.java

```diff
@@ -330,7 +330,8 @@ protected void doPost(HttpServletRequest httpServletRequest,
     private void processData(CohortDetails cohortDetails,
                              String geneList,
 							 HashSet<String> geneticProfileIdSet,
-							 String sampleSetId, String sampleIds,
+							 String sampleSetId, 
+                             String sampleIdsStr, //raw string from "user custom case ids" box
```

Comment:
I'm confused. Why don't we process the sampleIdsStr outside of processData and make processData only accept sample arrary?